spec/system/cytoskeletal_form_submission_spec.rb

```diff
@@ -77,9 +77,15 @@
 
       # Select Additional Metadata Tab
       click_on "v-pills-additional-tab"
+
+      # Related Objects
       fill_in "related_identifier_1", with: related_identifier
       find("#related_identifier_type_1").find(:xpath, "option[@value='#{related_identifier_type}']").select_option
       find("#relation_type_1").find(:xpath, "option[@value='#{relation_type}']").select_option
+      click_on "Add Another Related Object"
+      fill_in "related_identifier_2", with: "https://doi.org/10.7554/eLife.52482"
+      find("#related_identifier_type_2").find(:xpath, "option[@value='DOI']").select_option
+      find("#relation_type_1").find(:xpath, "option[@value='#{relation_type}']").select_option
```

Comment:
```suggestion
      find("#relation_type_1").find("option[value='#{relation_type}']").select_option
```
I don't think we need the xpath syntax here, though it can be useful.